content/docs/2.12/concepts/admission-webhooks.md

```diff
@@ -15,3 +15,7 @@ KEDA will block all incoming changes to `ScaledObject` that don't match these ru
 - The scaled workload (`scaledobject.spec.scaleTargetRef`) is already autoscaled by another other sources (other ScaledObject or HPA).
 - CPU and/or Memory trigger are used and the scaled workload doesn't have the requests defined. **This rule doesn't apply to all the workload types, only to `Deployment` and `StatefulSet`.**
 - CPU and/or Memory trigger are **the only used triggers** and the ScaledObject defines `minReplicaCount:0`. **This rule doesn't apply to all the workload types, only to `Deployment` and `StatefulSet`.**
+
+KEDA will block all incoming changes to `TriggerAuthentication`/`ClusterTriggerAuthentication` that don't match these rules:
+
+- The Pod Identity has specified empty IdentityId.
```

Comment:
```suggestion
- The specified identity ID for Azure AD Workload Identity and/or Pod Identity is empty.
```

Reply:
That is fairly technical for conceptual docs though :) but it's ok for me.

Reply:
My afraid is that end users think that they have to set the identity id on each TriggerAuthentication. Maybe instead of my previous suggestion, a remark like:
"This only applies if TriggerAuthentication overrides the default identityId"or something like that. 
I want to keep clear that the field isn't required and it's only for overriding default value

Reply:
Addressed in other comment